packages/client/src/forms/index.ts

```diff
@@ -731,6 +731,10 @@ export interface IRedirectFormField extends IFormFieldBase {
   type: typeof REDIRECT
   options: {
     url: string
+    callback: {
+      trigger: string
+      params: Record<string, string>
+    }
```

Comment:
Good to comment/document here(/where?) on what these refer to exactly with examples